src/curl.cpp

```diff
@@ -1174,6 +1175,23 @@ bool S3fsCurl::SetProxyUserPwd(const char* file)
     return true;
 }
 
+bool S3fsCurl::SetIPResolveType(const char* value)
+{
+    if(!value){
+        return false;
+    }
+    if(0 == strcasecmp(value, "ipv4") || 0 == strcasecmp(value, "v4")){
+        S3fsCurl::ipresolve_type = CURL_IPRESOLVE_V4;
+    }else if(0 == strcasecmp(value, "ipv6") || 0 == strcasecmp(value, "v6")){
+        S3fsCurl::ipresolve_type = CURL_IPRESOLVE_V6;
+    }else if(0 == strcasecmp(value, "whatever") || 0 == strcasecmp(value, "both") || 0 == strcasecmp(value, "default")){
```

Comment:
Remove unneeded "both" value?  Also do we need `ipv4` and `ipv6`?  This will read weirdly: `ipresolve=ipv4`.

Reply:
This was an option value that I added casually, but it seems inappropriate.
Make it only ipv4/ipv6/whatever.

Reply:
Changed option values, please re-check it.